charts/nucliadb_writer/values.yaml

```diff
@@ -1,14 +1,31 @@
+# -----
 # NucliaDB Writer Settings
+# -----
+
+# Image settings
+imagePullPolicy: IfNotPresent
+containerRegistry: CONTAINER_REGISTRY_TO_REPLACE
+image: IMAGE_TO_REPLACE
+
+# Component settings
+# ---
+# These settings are controlled through environment variables
+# all kubernetes oriented settings are controlled through
+# other values.
+# HOWEVER, we are in a transition period where we allow
+# both until all component settings are removed from
+# settings not mapped directly through env
+env:
```

Comment:
Should this be named `component_settings` instead? or `env_settings`.
Not a strong opinion though, I am fine with `env` + the comment too

Reply:
hmmm, looking more at it, `env` also matches what github actions does so I think I'd like to keep it as the simple `env` if that's okay?